homeworks/editerdelyhorvath/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,67 @@
+'''
+3. házi feladat
+
+Feladat 2: Rock Paper Scissors
+(while és for loop használata)
+'''
+
+from terminal_clearer import clear_terminal
+import random
+
+clear_terminal()
+
+while True:
+    # Ask the user for input
+    turns_nr = int(input("How many rounds would you like to play in Rock Paper Scissors? (Enter an odd number): ").strip())
+
+    # Check if the number is odd
+    if turns_nr % 2 != 0:
+        break
+    else:
+        print("That's an even number, try again.")
+
+# Tuple to store the options
+rock_paper_scissors_options = ("Rock", "Paper", "Scissors")
+
+player_score = 0
+computer_score = 0
+tie = 0
+
+for turn in range(1, turns_nr + 1):
+    print(f"Turn {turn}.")
+
+    # Computer makes a random choice
+    computer_choice = random.choice(rock_paper_scissors_options)
+
+    # Ask the user for their choice
+    while True:
+        # Ask the user for input
+        print(f"Choose your option: \n1: Rock \n2: Paper\n3: Scissors")
+        user_input = int(input("Enter 1, 2, or 3: "))
+        # Check if the user_input is 1,2 or 3
+        if user_input == 1 or user_input == 2 or user_input == 3:
+            break
+        else:
+            print("Not an appropriate choice, try again.")
+
+
+    # Convert user input to the corresponding option
+    user_choice = rock_paper_scissors_options[user_input - 1]
+
+    # Print the choices
+    print(f"You chose: {user_choice} \nThe computer chose: {computer_choice}\n\n")
+
+    # Determine the winner
+    if user_choice == computer_choice:
+        print("It's a tie!")
```

Comment:
A tie-okat szuper lenne ha nem engednénk tovább, hogy a végkimenetel se lehessen döntetlen. 
Ezt megoldhatod úgy, hogy a for loop helyett egy while loop-ot használsz, előtte bevezetve egy current_turn változót aminek az értékét növeled a loop végén. Valami ilyesmi kerül a for loop helyére:
`while current_turn <= turns_nr:`
Ha viszont döntetlen van, egyből megismétled a jelenlegi kört` continue`-val.

Reply:
**Szia István,**

Köszönöm szépen a kommenteket. Végig javítottam a kódjaimat terminal_clearer megfelelő importálásával.

A kő papír olló játék nekem így tetszett, azért írtam meg ilyenre. Kifejezetten örült a kislányom, amikor 2 nyerése és egy döntetlenje volt. De meghajtottam a fejem a megrendelő óhaja előtt és implementáltam a kért fícsört :) .

Reply:
Szia! Köszi szépen! :) 
Nyomsz kérlek egy commit, push-t hogy meg tudjam nézni a változtatásokat és jóvá tudjam hagyni?

Köszi!

Reply:
Még küzdök a mozi jegyfoglalóval, utána commit-olok :), köszönöm a türelmed.

Reply:
@NagyIstvanGabor , megvan a commit és a push. 

Még tudtam volna játszani a jegyfoglalóval, de tudni kell abbahagyni is... A festészetben legalábbis ez egy fontos pont.

Még ki lehetett volna vinni a mátrixot egy másik file-ba, lehetne több mátrix az egyes filmekhez... Lehetnének a filmek is másik file-ban definiálva... 

A lehetőségek tárháza, de úgy érzem a pillanatnyi mentett állapotom (beküldött megoldásom) lefedi a kért kritériumokat. 

Köszönöm.